motionplan/tpSpaceRRT_test.go

```diff
@@ -57,45 +57,6 @@ func TestPtgRrtBidirectional(t *testing.T) {
 	plan, err := tp.plan(ctx, goalPos, nil)
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, len(plan), test.ShouldBeGreaterThanOrEqualTo, 2)
-
-	allPtgs := ackermanFrame.(tpspace.PTGProvider).PTGSolvers()
-	lastPose := spatialmath.NewZeroPose()
-
-	if pathdebug {
-		for _, mynode := range plan {
-			trajPts, _ := allPtgs[int(mynode.Q()[0].Value)].Trajectory(mynode.Q()[1].Value, mynode.Q()[2].Value)
-			for i, pt := range trajPts {
-				intPose := spatialmath.Compose(lastPose, pt.Pose)
-				if i == 0 {
-					tp.logger.Debugf("$WP,%f,%f", intPose.Point().X, intPose.Point().Y)
-				}
-				tp.logger.Debugf("$FINALPATH,%f,%f", intPose.Point().X, intPose.Point().Y)
-				if i == len(trajPts)-1 {
-					lastPose = intPose
-					break
-				}
-			}
-		}
-	}
-	tp.planOpts.SmoothIter = 20
-	plan = tp.smoothPath(ctx, plan)
-	if pathdebug {
-		lastPose = spatialmath.NewZeroPose()
-		for _, mynode := range plan {
-			trajPts, _ := allPtgs[int(mynode.Q()[0].Value)].Trajectory(mynode.Q()[1].Value, mynode.Q()[2].Value)
-			for i, pt := range trajPts {
-				intPose := spatialmath.Compose(lastPose, pt.Pose)
-				if i == 0 {
-					tp.logger.Debugf("$SMOOTHWP,%f,%f", intPose.Point().X, intPose.Point().Y)
-				}
-				tp.logger.Debugf("$SMOOTHPATH,%f,%f", intPose.Point().X, intPose.Point().Y)
-				if pt.Dist >= mynode.Q()[2].Value {
-					lastPose = intPose
-					break
-				}
-			}
-		}
-	}
 }
 
 func TestPtgRrtUnidirectional(t *testing.T) {
```

Comment:
Is there a reason for this test anymore?

Reply:
There is not; removed.